client/verta/verta/registry/entities/_modelversion.py

```diff
@@ -1690,3 +1692,38 @@ def del_dataset_version(self, key):
 
         del self._msg.datasets[ind]
         self._update(self._msg, method="PUT")
+
+    def list_builds(self) -> List[Build]:
+        """
+        Gets this model version's past and present builds.
+
+        Builds are returned in order of creation (most recent first).
+
+        Returns
+        -------
+        list of :class:`~verta.endpoint.build.Build`
+
+        Examples
+        --------
+        To fetch builds that have passed their scans:
+
+        .. code-block:: python
+
+            passed_builds = list(filter(
+                lambda build: build.get_scan().passed,
+                model_ver.list_builds(),
+            ))
+
+        To fetch builds that haven't been scanned in a while:
+
+        .. code-block:: python
+
+            from datetime import datetime, timedelta
+
+            past_builds = list(filter(
+                lambda build: build.get_scan().date_updated < datetime.now().astimezone() - timedelta(days=30),
+                model_ver.list_builds(),
+            ))
+
+        """
```

Comment:
![Screen Shot 2023-04-20 at 4 14 16 PM](https://user-images.githubusercontent.com/96442646/233505896-f8259c57-2d96-4ac5-9a53-d9d18082a77e.png)

Reply:
it's cool we can do all this filtering now!!

Reply:
Nice work on these examples!